docker/Dockerfile-imgops

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:14.04
```

Comment:
Not too sure what image is the best to use here - `debian:jessie` would work too. The main consideration/concern is the size of the base image to download.